client/state/selectors/get-request.js

```diff
@@ -0,0 +1,22 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { getRequest, getRequestKey } from 'state/data-layer/wpcom-http/utils';
+
+/**
+ * Returns meta information about data requests going through the data layer
+ *
+ * @param {Object} state Redux state
+ * @param {Object} action data request action
+ * @returns {*} metadata about request
+ */
+export default ( state, action ) => {
+	const data = getRequest( state, getRequestKey( action ) );
+
+	return {
+		...data,
+		hasLoaded: data.lastUpdated > -Infinity,
```

Comment:
this is absurdly nitty, but `-Infinity` looks silly considering we don't have a notion of negative time...in this universe at least.  

would `hasLoaded: isNumber( data.lastUpdated )` work just as well?